components/Features/Features.tsx

```diff
@@ -0,0 +1,41 @@
+import Button from './../Button/Button';
+import { Icons } from './constants';
```

Comment:
I don't think Icon is a constant. It is an asset.

Reply:
@oguzergul  It think said that like it, It's going to be change

Reply:
I did not understand your reply @erdemkarakas what did I say?